vividus-plugin-rest-api/build.gradle

```diff
@@ -3,6 +3,7 @@ project.description = 'Vividus plugin for REST API testing'
 dependencies {
     api project(':vividus-bdd-engine')
     implementation project(':vividus-http-client')
+    implementation project(':vividus-plugin-datetime')
```

Comment:
where is this dependency used?

Reply:
<bean id="dateValidationSteps" class="org.vividus.bdd.steps.DateValidationSteps">
        <property name="softAssert" ref="jsonSoftAssert" />
    </bean>
used in 
    <util:list id="stepBeanNames-Api" value-type="java.lang.String">
        <value>apiSteps</value>
        <value>connectionValidationSteps</value>
        <value>httpResponseValidationSteps</value>
        <value>httpResponseSteps</value>
        <value>jsonResponseValidationSteps</value>
        <value>dateValidationSteps</value>
    </util:list>

Reply:
steps bean declaration should be moved to `vividus-plugin-datetime` (just create a new list for this plugin) and dependency could be removed